packages/mask/src/plugins/Tips/SNSAdaptor/TipsEntranceDialog.tsx

```diff
@@ -0,0 +1,359 @@
+            return
+        }
+        walletsList.forEach((x, idx) => {
+            x.isPublic = 1
+            x.isDefault = 0
+            if (idx === 0) {
+                x.isDefault = 1
+                return
+            }
+        })
+        setRawWalletList(cloneDeep(walletsList))
+        setRawPatchData(cloneDeep(walletsList))
+    }, [proofRes, kv, bodyViewStep])
+
+    const onCancel = () => {
+        setRawPatchData(cloneDeep(rawWalletList))
+        setHasChanged(false)
+    }
+
+    const refresh = () => {
+        setBodyViewStep(BodyViewStep.Main)
+        retryProof()
+        retryKv()
+    }
+    const titleTail = (
+        <WalletButton
+            step={bodyViewStep}
+            onClick={() => {
+                setBodyViewStep(bodyViewStep === BodyViewStep.Wallets ? BodyViewStep.AddWallet : BodyViewStep.Wallets)
+            }}
+        />
+    )
+    const setAsDefault = (idx: number) => {
+        const changed = cloneDeep(rawPatchData)
+        changed.forEach((x: any) => (x.isDefault = 0))
+        changed[idx].isDefault = 1
+        const defaultItem = changed[idx]
+        changed.splice(idx, 1)
+        changed.sort((a, b) => Number.parseInt(b.last_checked_at, 10) - Number.parseInt(a.last_checked_at, 10))
+        changed.unshift(defaultItem)
+        setRawPatchData(changed)
+        setHasChanged(true)
+    }
+
+    const onSwitchChange = (idx: number, v: boolean) => {
+        const changed = cloneDeep(rawPatchData)
+        changed[idx].isPublic = v ? 1 : 0
+        setRawPatchData(changed)
+        setHasChanged(true)
+    }
+
+    const [kvFetchState, onConfirm] = useAsyncFn(async () => {
+        try {
+            const payload = await getKvPayload(rawPatchData)
+            if (!payload || !payload.val) throw new Error('payload error')
+            const signResult = await Services.Identity.generateSignResult(
+                currentPersonaIdentifier as ECKeyIdentifier,
+                (payload.val as NextIDStoragePayload).signPayload,
+            )
+            if (!signResult) throw new Error('sign error')
+            await kvSet(payload.val, signResult.signature.signature, rawPatchData)
+            showSnackbar('Persona signed successfully.', {
+                variant: 'success',
+                message: nowTime,
+            })
+            retryProof()
+            retryKv()
+            return true
+        } catch (error) {
+            showSnackbar('Persona Signature failed.', {
+                variant: 'error',
+                message: nowTime,
+            })
+            return false
+        }
+    }, [hasChanged, rawPatchData])
+    const { setDialog } = useRemoteControlledDialog(WalletMessages.events.selectProviderDialogUpdated)
+    const onConnectWalletClick = useCallback(() => {
+        if (account) {
+            setBodyViewStep(BodyViewStep.AddWallet)
+        } else {
+            setDialog({
+                open: true,
+                pluginId: PluginId.Tip,
```

Comment:
To follow the folder name. 

```suggestion
                pluginId: PluginId.Tips,
```